spec/factories/middleware_domain.rb

```diff
@@ -1,10 +0,0 @@
-FactoryBot.define do
```

Comment:
We're deleting this factory but not the middleware_domain model?

Reply:
Removed.